packages/flutter/lib/src/painting/box_decoration.dart

```diff
@@ -79,8 +80,15 @@ class BoxDecoration extends Decoration {
     this.borderRadius,
     this.boxShadow,
     this.gradient,
+    this.backgroundBlendMode,
     this.shape = BoxShape.rectangle,
-  }) : assert(shape != null);
+  }) : assert(shape != null),
+       // TODO(mattcarroll): Use "backgroundBlendMode == null" when Dart #31140 is in.
```

Comment:
replace "Dart #31140" with a link to the issue

Reply:
done